ERCS/erc-7439.md

```diff
@@ -0,0 +1,272 @@
+---
+eip: 7439
+title: Prevent ticket touting.
+description: An interface for customers to resell their tickets via authorized ticket resellers.
+author: LeadBest Consulting Group <service@getoken.io>, Sandy Sung (@sandy-sung-lb), Mars Peng <mars.peng@getoken.io>, Taien Wang <taien.wang@getoken.io>
+discussions-to: https://ethereum-magicians.org/t/prevent-ticket-touting/15269
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-07-28
+requires: 165, 721
```

Comment:
You don't explain how these proposals are supposed to be used in your specification section. For ERC-165, I'd add a note like:

> Contracts implementing this proposal MUST return true when ERC-165's `supportsInterface` is called with `0xYourInterfaceId`.

For ERC-721, you could maybe add this to your code block:

> ```solidity
> interface IERC7439 /* is ERC721 */ {
> ```